client/js/components/SmallUnsubscribeButton/SmallUnsubscribeButton.js

```diff
@@ -0,0 +1,13 @@
+import React, {Component} from 'react';
+import styles from './styles';
+import {TouchableOpacity, Text} from 'react-native';
+
+const SmallUnsubscribeButton = props => {
+  return (
+    <TouchableOpacity style={styles.button}>
+      <Text style={styles.buttonText}>Unsubscribe</Text>
+    </TouchableOpacity>
+  );
+};
+
```

Comment:
Can you create a button that can toggle between subscribe and unsubscribe?

Reply:
Hey Sid, that’s not a part of the Design Requirements.

Reply:
What do you mean? slack me.